app/config.py

```diff
@@ -22,7 +22,7 @@ class Config(object):
     SES_ARN = os.environ.get("SES_ARN")
     SES_SENDER = os.environ.get("SES_SENDER")
     METRICS_EMAIL_ADDRESS = os.environ.get("METRICS_EMAIL_ADDRESS", "metrics@sparc.science")
-    TESTING_EMAIL_ADDRESS = os.environ.get("TESTING_EMAIL_ADDRESS", "jessekhora@gmail.com")
+    TESTING_EMAIL_ADDRESS = os.environ.get("TESTING_EMAIL_ADDRESS")
```

Comment:
RIP Jesse :( who's email address should we use now? There is nothing set in the env vars on Heroku so it was using the default

Reply:
It does not need to be set up on staging / production server. It is only used when running the tests.

Reply:
oh right, but regardless we will still need to have one set right?

Reply:
The email address has been set on our Jenkins test instances. I would not advice setting up a default value, the test will send an email to the specified address and it will not be helpful if it is not set to the email address of the person running the tests .

Reply:
ah ok, just wanted to make sure it was set somewhere. Lgtm!